site/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "@blockprotocol/site",
-  "version": "0.1.0",
-  "description": "The public-facing blockprotocol.org website",
-  "repository": "https://github.com/blockprotocol/blockprotocol.git",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
This will help us avoid accidental publishing to NPM. I removed description and repository because they are not practically useful for non-punishable packages.